sw/device/tests/BUILD

```
@@ -511,6 +512,53 @@ opentitan_functest(
     ],
 )
 
+otp_json(
+    name = "otp_json_uncalibrated_clocks",
+    partitions = [
+        otp_partition(
+            name = "CREATOR_SW_CFG",
+            items = {
+                # Enable AST initialization.
+                "CREATOR_SW_CFG_AST_INIT_EN": otp_hex(CONST.MUBI4_FALSE)
+            },
+        ),
+    ],
+)
+
+otp_image(
+    name = "otp_uncalibrated_clocks",
+    verbose = True,
+    src = "//hw/ip/otp_ctrl/data:otp_json_test_unlocked0",
+    overlays = STD_OTP_OVERLAYS + [":otp_json_uncalibrated_clocks"],
```

If you want to emulate a completely empty OTP that is in the TEST_UNLOCKED0 lc_state, then you would just do:
```suggestion
    overlays = [":otp_json_uncalibrated_clocks"],
```
But this will not have anything in the SW_CREATOR_CFG partitions

---

That is okay... In fact, I want it to be practically empty.